sematic/config/settings.py

```diff
@@ -473,6 +478,16 @@ def _ensure_mandatory_plugins(plugin_settings: PluginsSettings) -> None:
             plugin_settings[plugin_path] = {}
 
 
+def _ensure_scoped_plugins(plugin_settings, scopes):
+    """
+    Adds any plugins that are in a scope to the parameter, if not present.
```

Comment:
Without this, env var overrides don't work--even if a plugin is listed in a scope